prql-compiler/tests/integration/main.rs

```diff
@@ -212,3 +214,15 @@ fn replace_booleans(rows: &mut Vec<Row>) {
         }
     }
 }
+
+// MySQL may adds 0s to the end of results of `/` operator
+fn remove_trailing_zeros(rows: &mut Vec<Row>) {
```

Comment:
This is almost cheating... The `/` operator in MySQL returns DECIMAL, which appends the 0. It does transparently convert to INTEGER when needed, at least in all of my testing.

If sometime later we find that MySQL has a problem related to decimals, this is it.

Reply:
I don't know if we should dare to use the less accurate FLOAT type.
Shouldn't the comparison be based on the accuracy of the actual numbers rather than the display method, and if the issue here is only display, then why not just use the DECIMAL type?

Reply:
Is it a decimal vs a float, or a decimal vs an integer?

The float / decimal distinction seems less significant (though I don't have much experience with decimals...)

Reply:
And we can always reduce the support level of some of these dialects if they're taking time away from other things — MySql & MsSql are top of the chopping block!

Reply:
Personally, I think it would be great if MySQL is supported since ClickHouse and Detabend are MySQL compatible.
(By the way I just found out today that Detabend has the `//` operator. Haven't checked to see how it works.)

I feel that MSSQL has a significant disadvantage to support because it cannot use `USING`...

Reply:
Oh, I though this was int vs decimal, but it's float vs decimal. My bad.

This is the related rule, (probably) https://github.com/PRQL/prql/pull/2720/files#diff-f6d964f7215b03bfdb696d81cf4258db7cf5686558bf44fa627d031f2f474c2eR125

The more experimentation I did, the more I find that MySQL prefers using decimals over floats. Nothing wrong with that, but it's different than most other engines.

(decimals are basically fixed-point numbers. stored as integers + the point offset)

Well in anycase, this should not be a problem in the near future, just something to keep in mind.